tests/dataimportcron_test.go

```diff
@@ -48,6 +48,7 @@ var _ = Describe("DataImportCron", func() {
 		ns                  string
 		scName              string
 		originalProfileSpec *cdiv1.StorageProfileSpec
+		oldDvName           = "src-garbage-0"
```

Comment:
oldDvName should move into the function, as nobody else is using it

Reply:
Okay